test/Sentry.Tests/Extensibility/DisabledHubTests.cs

```diff
@@ -15,7 +15,7 @@ public class DisabledHubTests
 
         [Fact]
         public void ConfigureScopeAsync_ReturnsCompletedTask()
-            => Assert.Equal(default, DisabledHub.Instance.ConfigureScopeAsync(null));
+            => Assert.Equal(Task.CompletedTask, DisabledHub.Instance.ConfigureScopeAsync(null!));
```

Comment:
null! ?

Reply:
just to quiet the warning, it doesn't matter here